fjord/settings/base.py

```diff
@@ -581,10 +582,12 @@ def lazy_langs():
 # function handles the extraction.
 DOMAIN_METHODS = {
     'django': [
-        ('fjord/**.py', 'fjord.base.l10n.extract_python'),
-        ('fjord/**/templates/**.html', 'fjord.base.l10n.extract_template'),
-        ('fjord/**/jinja2/**.html',  'fjord.base.l10n.extract_template'),
-        ('templates/**.html', 'fjord.base.l10n.extract_template'),
+        ('fjord/**.py', 'babel.messages.extract.extract_python'),
+        ('fjord/**/jinja2/**.html',  'jinja2.ext.babel_extract'),
+        # FIXME: These are using Jinja2 template extraction for Django
+        # templates. Does that work right?
+        ('fjord/**/templates/**.html', 'jinja2.ext.babel_extract'),
+        ('templates/**.html', 'jinja2.ext.babel_extract'),
```

Comment:
We should provide helpers for these because otherwise they're hard to remember. Maybe something like:

* `fjord.l10n.extractors.extract_python`
* `fjord.l10n.extractors.extract_jinja2`
* `fjord.l10n.extractors.extract_django`